packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/index.tsx

```diff
@@ -95,13 +107,26 @@ export const Dependencies: FunctionComponent = () => {
               }
             />
           ))}
-          <AddResource
+          <WorkspaceSubtitle>Google Fonts</WorkspaceSubtitle>
+
+          <AddFont
             addResource={resource =>
               workspace.externalResourceAdded({
                 resource,
               })
             }
           />
+          {fonts.map(resource => (
+            <ExternalFonts
```

Comment:
This should use list elements, for accessibility

Reply:
Good point! Done :)